content/community/community-team/engagement-channels/contents.lr

```diff
@@ -0,0 +1,173 @@
+_model: page
+---
+_template: page-with-toc.html
+---
+title: Community Team: Engagement Channels
+---
+description: This page serves as a guide for Community Team members on how to engage with CC Open Source across various channels.
+---
+is_long_page: true
+---
+body:
+
+You've joined the CC Community Team. Congratulations! This is a very clear
+indication of your desire to engage more deeply with the Creative Commons
+community. As a fully remote organisation, CC uses a lot of tools to stay
+organised and collaborate. As you join the Community Team, depending on your
+role, you will be granted different levels of access to these tools and
+software.
+
+Given the breadth and diversity of these tools coupled with varying levels of
+access, it can be easy to not know what is expected of you and how you can make
+the most of your involvement. This guide will condense this information down to
+help you identify the best ways to engage with the team. _Let's get started!_
+
+## Meetings
+
+We regularly hold meetings to discuss work and, occasionally, to play games.
+Based on your role and the project, you will be invited to some of these
+meetings. You can join the CC Engineering calendar to keep up with the schedule.
+
+<!-- TODO: describe meetings here as they are finalized -->
+
+## Slack
+
+We communicate over text using Slack. Everyone from the community is on it,
+including CC employees. You can find fellow developers, technical writers and
+designers all in the same place. If you are, or want to be, a part of the
+community, [join us on Slack](/community/#slack). It is one of the best ways to
+connect with us.
+
+Slack is organised by channels and, depending on your role, you will be invited
+to these channels to participate in discussions about certain topics.
+
+### Newbie channels
+
+There are a number of channels which budding open source developers join in
+hopes of finding like-minded developers. You should help them get started, clear
+simple doubts, introduce them to FOSS and make them feel welcome in our
+community. After all, the more the merrier!
+
+**Channel:** `#general`, `#cc-dev-internships`, `#cc-developers`
+
+### Community team channel
+
+This is a channel where you can connect with other members of the Community
+Team. Announcements pertaining to the Community Team and planned changes to the
+community will be announced here first.
+
+**Channel:** `#cc-dev-community-team`
+
+### Engineering channel
+
+This is a channel where the entire engineering team of CC can be reached.
+
+**Channel:** `#engineering`
+
+### UX design channel
+
+This is the channel where we discuss design and UI/UX of various CC software and
+websites. This is particularly relevant to the Vocabulary project.
+
+**Channel:** `#ux-design`
```

Comment:
we also have ux-design-figma for Figma notifications

Reply:
Added #ux-design-figma in 430235b.